api/graphql/schema.graphql

```diff
@@ -447,6 +447,11 @@ enum ApplicationStep {
   REVIEW_AND_SUBMIT
 }
 
+enum ApplicationCriteria {
```

Comment:
I'm not sure about the name of this. I feel like it should mention education. Or "minimum experience" maybe?

Reply:
Sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/6379/commits/64e7e7fde21402c448472ac0d2172fdcbaca9cb9

Reply:
I see you've got minimumCriteria in a number of places through the branch.  Those should probably be minimumExperienceCriteria, right?  It should always be experience + criteria since criteria on its own doesn't mean much.

... I think.  Naming is hard.  :laughing: